tokio/src/util/linked_list.rs

```diff
@@ -243,6 +229,53 @@ impl<L: Link> fmt::Debug for LinkedList<L, L::Target> {
     }
 }
 
+// ===== impl CountedLinkedList ====
+
+// Delegates operations to the base LinkedList implementation, and adds a counter to the elements
+// in the list.
+pub(crate) struct CountedLinkedList<L: Link, T> {
+    _list: LinkedList<L, T>,
+    _count: AtomicUsize,
```

Comment:
These fields are not unused, so their names should not start with `_`.
```suggestion
    list: LinkedList<L, T>,
    count: AtomicUsize,
```